### ISSUE TITLE
docs: create Accessibility page within Documentation

### ISSUE THREAD

**Issue by ilhan007** [SAP/ui5-webcomponents#3652]
### **Issue Description**
The Documentation lacks information regarding the Accessibility support the UI5 Web Components provide.
See https://github.com/SAP/ui5-webcomponents/issues/3647

### Points to describe
Create a Accessibility page, under Documentation, to provide information about:
- the standard we follow (Web Content Accessibility Guidelines 2.1 Level AA)
- Screen Readers we test on (Jaws + Chrome)
- Keyboard Handling we provide
- the High Contrast Themes we have (Black and White)
- the contrast ratio on all themes for better experience.
- other?

### **Issue Type**

- [ ] Documentation is unclear
- [ ] Documentation is incorrect
- [x] Documentation is missing
- [ ] Other
